### PR TITLE
removed new logins for year 15 students

### PR DESCRIPTION
**Pull request by subodhk01**

### CODE REVIEW

File: myapp/views.py [hunk 1]
```diff
@@ -114,6 +114,14 @@ def profile(request):
         query = r"(?<=\.)(.*)(?=\@)"
         branch = str(re.search(r"(?<=\.)(.*)(?=\@)", email).group(0))[-5:-2]
         if(IDDyear+"@" in email):
+            ####### Temporary
+            user = User.objects.get(email=email)
+            try:   
+                r = requests.get('http://kyukey-lock.herokuapp.com/a2oj/'+ str(email))
```
Comment (shivanshs9) [inline myapp/views.py:120]: Why this?

File: myapp/views.py [hunk 2]
```diff
@@ -114,6 +114,14 @@ def profile(request):
...
+            except:
+                pass
+            return redirect('logout')
```
Comment (shivanshs9) [inline myapp/views.py:123]: This is just gonna logout all IDD peeps whenever they visit profile page.
Plus, hackers can always disable redirection and hence free to explore yearbook

File: myapp/views.py (hunk 1 above)
Comment (subodhk01) [reply myapp/views.py:120]: Just to log all the new emails.

File: myapp/views.py (hunk 2 above)
Comment (subodhk01) [reply myapp/views.py:123]: Not all the IDD students, only the new ones.

Comment (subodhk01) [reply myapp/views.py:123]: They won't be able to explore other pages as they'll get server error as there is no student linked to their User.

File: myapp/views.py (hunk 1 above)
Comment (shivanshs9) [reply myapp/views.py:120]: It always returns success for any email, and if you just want to log you can print to a file. Seriously why a network call to a2oj endpoint for logging?

File: myapp/views.py [hunk 3]
```diff
@@ -114,6 +114,14 @@ def profile(request):
         query = r"(?<=\.)(.*)(?=\@)"
         branch = str(re.search(r"(?<=\.)(.*)(?=\@)", email).group(0))[-5:-2]
         if(IDDyear+"@" in email):
+            ####### Temporary
+            user = User.objects.get(email=email)
```
Comment (shivanshs9) [inline myapp/views.py:118]: You aren't using user instance later either

File: myapp/views.py (hunk 2 above)
Comment (shivanshs9) [reply myapp/views.py:123]: While it isn't a recommended way, it's fine. But atleast do a better way to log (like in a file) and then redirect to logout

File: myapp/views.py (hunk 3 above)
Comment (subodhk01) [reply myapp/views.py:118]: Actually I was thinking of deleting the user earlier. I'll fix that

File: myapp/views.py [hunk 4]
```diff
@@ -110,10 +111,16 @@ def profile(request):
     try:
         u = request.user.student
     except:
-        IDDyear = "15"
+        IDDyear = "19"
```
Comment (shivanshs9) [inline myapp/views.py:114]: Ye change undo karna h?

Comment (subodhk01) [reply myapp/views.py]: shit sorry..

Comment (subodhk01) [reply myapp/views.py]: Done

**Pull request merged**